src/gen/conditions.rs

```diff
@@ -282,8 +282,15 @@ pub fn parse_args(
         }
         ASSERT_SECONDS_RELATIVE => {
             maybe_check_args_terminator(a, c, flags)?;
-            let seconds = parse_seconds(a, first(a, c)?, ErrorCode::AssertSecondsRelative)?;
```

Comment:
`parse_seconds()` returns a `u64` and will return 0 if the value is negative (to indicate a no-op). With this patch, we want to preserve a condition of 0, and distinguish it from a negative (no-op) value.